components/usage/pkg/controller/reconciler.go

```diff
@@ -131,7 +131,7 @@ func (u *UsageReconciler) ReconcileTimeRange(ctx context.Context, from, to time.
 func generateUsageReport(teams []teamWithWorkspaces, maxStopTime time.Time) ([]TeamUsage, error) {
 	var report []TeamUsage
 	for _, team := range teams {
-		var teamTotalRuntime float64
+		var teamTotalRuntime int64
```

Comment:
Thought: This could even be `uint64` right? (I don't think we want to support "negative usage time", so we can double our max representable value 🙌)

Reply:
If this really is only about "usage"/"runtime", I agree. If we want to represent "credits" here (which I'm not sure we want/should..?) then int64 it is.
Another thought: Not sure how (type) safe `uint64`/`int64` interactions are in Go: Is it easy to mess it up? If yes, that might be a good argument too keep one type everywhere (most likely `in64` if we need the "positive" side)